Access review v2 api (#9935)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/accessreviewinstance-sendreminder.md

---

> title: "SendReminder accessReviewInstance"

Reviewer: accessReviewInstance: sendReminder

> description: "In the Azure AD access reviews feature, send a reminder to the reviewers of a currently active accessReviewInstance."

Reviewer: ```suggestion
description: "Sends a reminder to the reviewers of a currently active accessReviewInstance."
```

localization_priority: Normal
author: "isabelleatmsft"
ms.prod: "microsoft-identity-platform"
doc_type: apiPageType
---

> # SendReminder accessReviewInstance

Reviewer: accessReviewInstance: sendReminder

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> In the Azure AD [access reviews](../resources/accessreviewsv2-root.md) feature, send a reminder to the reviewers of a currently active [accessReviewInstance](../resources/accessreviewinstance.md).

Reviewer: ```suggestion
Send a reminder to the reviewers of a currently active [accessReviewInstance](../resources/accessreviewinstance.md).
```

## Permissions
One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).

|Permission type                        | Permissions (from least to most privileged)              |
|:--------------------------------------|:---------------------------------------------------------|

> |Delegated (work or school account)     | AccessReview.ReadWrite.All |

Reviewer: Add this row:

|Delegated (personal Microsoft account)|Not supported.|

|Application                            | AccessReview.ReadWrite.All |

> ## HTTP request

Reviewer: Remove line 26.

Author: Wondering why? all other files have this line

Reviewer: Ignore comment.

<!-- { "blockType": "ignored" } -->
```http
POST /identityGovernance/accessReviews/definitions/{definitionId}/instances/{instanceId}/sendReminder
```
## Request headers
None.

## Request body
Do not supply a request body for this method.

## Response
If successful, this method returns a `204, No Content` response code. It does not return anything in the response body.

> ## Example

Reviewer: ```suggestion
## Examples
```

> ##### Request

Reviewer: This should be a H3, not an H4: ### Request

> # [HTTP](#tab/http)

Reviewer: Remove line 43.

<!-- {
  "blockType": "request",
  "name": "sendReminder_accessReview"
}-->
```http
POST https://graph.microsoft.com/beta/identityGovernance/accessReviews/definitions/04e5c3b2-9db2-40d3-a204-128f4956ae8e/instances/70463350-742e-4909-bfa5-bc23447bd002/sendReminder
```

> # [C#](#tab/csharp)

Reviewer: Remove lines 51 through 60.

[!INCLUDE [sdk-documentation](../includes/snippets/snippets-sdk-documentation-link.md)]

# [JavaScript](#tab/javascript)
[!INCLUDE [sdk-documentation](../includes/snippets/snippets-sdk-documentation-link.md)]

# [Objective-C](#tab/objc)
[!INCLUDE [sdk-documentation](../includes/snippets/snippets-sdk-documentation-link.md)]

---

> ##### Response

Reviewer: This should be a H3, not an H4: ### Response
